packaging/msi/SumoLogic.wixext/SumoLogicTests/ConfigUpdaterTests.cs

```diff
@@ -44,12 +55,39 @@ public void InstallationTokenAssertions(Config config, StreamReader sr)
             Assert.AreEqual("yaz", collectorFields.Children["xaz"]);
         }
 
+        public void OpAmpAssertion(Config config, StreamReader sr)
+        {
+            YamlStream ys = new YamlStream();
+            ys.Load(sr);
+            YamlMappingNode root = (YamlMappingNode)ys.Documents[0].RootNode;
+
+            Assert.IsTrue(root.Children.ContainsKey("extensions"));
+            Assert.AreEqual(YamlNodeType.Mapping, root.Children["extensions"].NodeType);
+            var extensions = (YamlMappingNode)root.Children["extensions"];
+
+            Assert.IsTrue(extensions.Children.ContainsKey("opamp"));
+            Assert.AreEqual(YamlNodeType.Mapping, extensions.Children["opamp"].NodeType);
+            var opamp = (YamlMappingNode)extensions.Children["opamp"];
+
+            Assert.IsTrue(opamp.Children.ContainsKey("remote_configuration_directory"));
+            Assert.AreEqual(YamlNodeType.Scalar, opamp.Children["remote_configuration_directory"].NodeType);
+            Assert.AreEqual(config.OpAmpFolder, opamp.Children["remote_configuration_directory"].ToString());
+
+            Assert.IsTrue(root.Children.ContainsKey("service"));

```

Comment:
do we need this service assertion as part of the OpAMP configuration test? My understanding here is that this is already a part of the original test configuration.